massgov/mayflower
# React/dp 10678  tabs a11y aria label

**Description:** 
This pr addresses a11y issues found in the mass.gov search a11y audit by @ygannett related to the tabs([DP-10678](https://jira.mass.gov/browse/DP-10678))
- adds passable aria-label propert to the tabs molecule

**Other Details:**
- moves tabs scss into the assets folder

**To test:**
1. pull branch
2. cd react
3. npm install
4. npm start
5. go to the tabs molecule story and confirm aria-label is passable
